modTxPool/src/org/aion/txpool/Constant.java

```diff
@@ -0,0 +1,6 @@
+package org.aion.txpool;
+
+public class Constant {
+
+    public static final int MAX_BLK_SIZE = 2 * 1024 * 1024;
+}
```

Comment:
The Constant class is part of a consensus for limiting the block size. Should not be part of txpool. Can move to the other place in the following modularity stuff. (This number can be set in the txpool during the init stage.)

Reply:
I will make a note of this on Jira. This was the only option for now, since `modAionImpl` and `modTxPoolImpl` both use this class but neither of them are dependent on each other, they're both dependent on `modTxPool` however.